calculator.rb

```diff
@@ -0,0 +1,64 @@
+# Calculator Project
+
+# Collect user input for first number
+puts "Welcome to calculator! Please enter the first number you'd like to use:"
+num_one = gets.chomp
+# Checks to see if user input is a number. Alerts user of incorrect input
+if num_one.to_i.to_s == num_one
+  num_one = num_one.to_f
+else
+  puts "\nYou did not enter a number. Try again."
+  exit
+end
+
+# Checks to see if user input is a number. Collect user input for second number
+puts "\nPlease enter the second number you'd like to use:"
+num_two = gets.chomp
+# Alerts user of incorrect input
+if num_two.to_i.to_s == num_two
+  num_two = num_two.to_f
+else
+  puts "\nYou did not enter a number. Try again."
```

Comment:
You could assign this message to a variable as well, so that you don't have to repeat it. ("As well" because I wrote this comment after the following one :) )